platforms/blabsy/src/components/tweet/tweet-stats.tsx

```
@@ -9,147 +9,155 @@ import { TweetShare } from './tweet-share';
+        setCurrentStats({
+            currentReplies: totalReplies,
+            currentLikes: totalLikes,
+            currentTweets: totalTweets
+        });
+    }, [totalReplies, totalLikes, totalTweets]);
 
-  const replyMove = useMemo(
-    () => (totalReplies > currentReplies ? -25 : 25),
-    [totalReplies]
-  );
+    const replyMove = useMemo(
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: currentReplies

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)